### DISCUSSION TITLE
Testing MetaEditor locally

### DISCUSSION THREAD

**Discussion by mateuszwojt** [markolofsen/metaeditor#8] [Q&A] [answered]
I'm a little bit confused because the documentation is incomplete. I have a couple of questions regarding the local testing procedure:

- What should I do in order to test the Pixel Streaming locally using the MetaEditor React front-end? 
- Does it work with the example signaling web server provided by Epic? 
- Do I need to use the MetaPlugin inside my Unreal project or does it work with the Epic's Pixel Streaming plugin as well?

**Comment by markolofsen** [comment, marked as answer]
* For local testing, you can use http://ps-dev.metaeditor.io/dev ... or launch MetaEditor on your localhost.
* MetaEditor is compatible with UE4 Signaling Server and is fully supported in UE5.
* You need to install the PixelStreaming plugin and MetaPlugin for it to work properly.

**Reply by mateuszwojt** [reply]
@markolofsen thanks for your answer. I'm still struggling though with the setup. I enabled both PixelStreaming and MetaEditor plugins, packaged my Unreal project, and run it together with the signaling server on localhost. But when I try to connect using http://ps-dev.metaeditor.io/dev, I'm immediately getting disconnected. Am I missing something?

Here's the suspicious messages from the console log:
```
NativeDOMDelegate.ts:749 Video Encoder QP can not be less then 0
n.onVideoEncoderAvgQP @ NativeDOMDelegate.ts:749
n.onDisconnect @ NativeDOMDelegate.ts:624
e.handleOnClose @ WebSocketController.ts:158
Logger.ts:22 Disconnected to the signalling server via WebSocket: 1008 - Unsupported message type
```

**Reply by markolofsen** [reply]
Try using MetaClient instead of Signaling Server.
https://metaeditor.io/docs/metaclient/description

**Comment by mjcc007** [comment]
Is the library still supported? http://ps-dev.metaeditor.io/dev is not accessible. The documentation is not detailed and the configuration items are not explained！ It's hard to get people to use. Can you improve the support later? thanks。
